blogposts/our-journey-to-all-remote.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Our journey to all-remote"
+author: Nick Snyder
+publishDate: 2020-03-30-T10:00-07:00
+tags: [
+  blog, remote
+]
+slug: our-journey-to-all-remote
+heroImage: /blog/sf-office-mural.jpg
+published: true
+---
+
+On February 20, 2020 we officially moved out of our amazing San Francisco office and became an [all-remote](https://about.sourcegraph.com/company/remote) company. Many people have not worked at an all-remote company before and I frequently get questions, especially from candidates, about what led to this decision, so this blog post documents our journey to all-remote.
+
+## Remote friendly engineering
+
+For most of Sourcegraph's history we have had an office in San Francisco, but many of our early engineering hires were distributed across the world so we have had remote-friendly practices ingrained in our culture from the very beginning. As a result, we strongly value things like asynchronous written communication, personal responsibility, and autonomy; nothing else works when your teammates are in different timezones.
+
+## Distributed days
+
+Since we were open to hiring engineers from all over the world, our engineering team organically grew to be more distributed over time. Continuing to be a remote-friendly place to work was important for us as we grew the team. In late 2018 we started experimenting with scheduled days where SF teammates were encouraged to work from a location other than the office; we called these "distributed days".
```

Comment:
```suggestion
Since we were open to hiring engineers from all over the world, our engineering team organically grew to be more distributed over time. Continuing to be a remote-friendly place to work was important for us as we grew the team. In late 2018, we started experimenting with scheduled days where SF teammates were encouraged to work from a location other than the office; we called these "distributed days".
```